src/main/java/com/synopsys/integration/jira/common/rest/service/PluginManagerService.java

```diff
@@ -29,9 +29,6 @@ public class PluginManagerService {
 
     public static final String API_PATH = "/rest/plugins/1.0/";
 
-    private static final String QUERY_KEY_OS_AUTH_TYPE = "os_authType";
```

Comment:
Why are we removing the auth type.  I believe this is validating that basic auth is being uses since we support other types.  We the parameter removed or replaced with a different query parameter?